docs/v3.x/plugins/users-permissions.md

```diff
@@ -860,3 +896,7 @@ module.exports = {
 ::: tip
 You can learn more on configuration in the documentation [here](../concepts/configurations.md)
 :::
+
+## Custom providers HOST URI (subdomain) configuration
+
+Some external Identity providers like AWS Cognito will require you to provide the link to the IDP host url (e.g. your_idp_name.auth.eu-central-1.amazoncognito.com) so that you can access the IDP login page using `htts://your_strapi_url/connect/your_custom_provider_name`. In order to do that you will have to fill in that value in the configuration popup of the provider in the Host URI (subdomain) field.
```

Comment:
Why did you add that? The doc for aws isn't enough?

Reply:
yes it is, it was added before the doc for AWS was written. Will remove it.